File: apps/hnnt-app/src/styles/colors.ts
// Design tokens for HNNT brand colors
export const colors = {
  // Primary brand colors
  primary: {
    50: '#FFF1F3',
    100: '#FFE4E8', 
    200: '#FFCDD6',
    300: '#FFA8B8',
    400: '#FF7396',
    500: '#FF4275', // Main brand color - vibrant pink
    600: '#E91E5E',
    700: '#C4124A',
    800: '#A31440',
    900: '#8B1538',
  },

  // Secondary/accent colors
  secondary: {
    50: '#F0F9FF',
    100: '#E0F2FE',
    200: '#BAE6FD',
    300: '#7DD3FC',
    400: '#38BDF8',
    500: '#0EA5E9', // Complementary blue
    600: '#0284C7',
    700: '#0369A1',
    800: '#075985',
    900: '#0C4A6E',
  },

  // Neutral grays
  neutral: {
    50: '#FAFAFA',
    100: '#F5F5F5',
    200: '#E5E5E5',
    300: '#D4D4D4',
    400: '#A3A3A3',
    500: '#737373',
    600: '#525252',
    700: '#404040',
    800: '#262626',
    900: '#171717',
  },

  // Semantic colors
  success: {
    50: '#F0FDF4',
    500: '#22C55E',
    600: '#16A34A',
  },
  
  warning: {
    50: '#FFFBEB',
    500: '#F59E0B',
    600: '#D97706',
  },
  
  error: {
    50: '#FEF2F2',
    500: '#EF4444',
    600: '#DC2626',
  },

  // Background colors
  background: {
    primary: '#FFFFFF',
    secondary: '#FAFAFA',
    tertiary: '#F5F5F5',
  },

  // Dark mode
  dark: {
    background: {
      primary: '#000000',
      secondary: '#0A0A0A',
      tertiary: '#171717',
    },
    text: {
      primary: '#FFFFFF',
      secondary: '#A3A3A3',
      tertiary: '#737373',
    }
  },

  // Social platform colors (from AuthButton)
  social: {
    google: '#4285F4',
    instagram: '#E1306C',
    snapchat: '#FFFC00',
    tiktok: '#000000',
    email: '#007AFF',
  },

  // Text colors
  text: {
    primary: '#171717',
    secondary: '#525252',
    tertiary: '#737373',
    inverse: '#FFFFFF',
  }
};

// Convenience exports for common use cases
export const brandColors = {
  primary: colors.primary[500],
  secondary: colors.secondary[500],
  background: colors.background.primary,
  text: colors.text.primary,
};

export default colors;

File: apps/ios/HINTO/Sources/Design/Colors.swift
import SwiftUI

extension Color {
    // MARK: - Primary Brand (Vibrant Pink)
    static let hintoPink = Color(hex: "FF4275")
    static let hintoPink50 = Color(hex: "FFF1F3")
    static let hintoPink100 = Color(hex: "FFE4E8")
    static let hintoPink200 = Color(hex: "FFCDD6")
    static let hintoPink300 = Color(hex: "FFA8B8")
    static let hintoPink400 = Color(hex: "FF7396")
    static let hintoPink600 = Color(hex: "E91E5E")
    static let hintoPink700 = Color(hex: "C4124A")
    static let hintoPink800 = Color(hex: "A31440")
    static let hintoPink900 = Color(hex: "8B1538")

    // MARK: - Secondary (Sky Blue)
    static let hintoBlue = Color(hex: "0EA5E9")
    static let hintoBlue50 = Color(hex: "F0F9FF")
    static let hintoBlue100 = Color(hex: "E0F2FE")
    static let hintoBlue400 = Color(hex: "38BDF8")

    // MARK: - Neutral
    static let neutral50 = Color(hex: "FAFAFA")
    static let neutral100 = Color(hex: "F5F5F5")
    static let neutral200 = Color(hex: "E5E5E5")
    static let neutral300 = Color(hex: "D4D4D4")
    static let neutral400 = Color(hex: "A3A3A3")
    static let neutral500 = Color(hex: "737373")
    static let neutral600 = Color(hex: "525252")
    static let neutral700 = Color(hex: "404040")
    static let neutral800 = Color(hex: "262626")
    static let neutral900 = Color(hex: "171717")

    // MARK: - Semantic
    static let hintoSuccess = Color(hex: "22C55E")
    static let hintoWarning = Color(hex: "F59E0B")
    static let hintoError = Color(hex: "EF4444")

    // MARK: - Social
    static let socialGoogle = Color(hex: "4285F4")
    static let socialInstagram = Color(hex: "E1306C")
    static let socialSnapchat = Color(hex: "FFFC00")
    static let socialTikTok = Color(hex: "000000")
    static let socialApple = Color(hex: "000000")
    static let socialFacebook = Color(hex: "1877F2")
}

extension Color {
    init(hex: String) {
        let hex = hex.trimmingCharacters(in: .alphanumerics.inverted)
        var int: UInt64 = 0
        Scanner(string: hex).scanHexInt64(&int)
        let a, r, g, b: UInt64
        switch hex.count {
        case 6:
            (a, r, g, b) = (255, (int >> 16) & 0xFF, (int >> 8) & 0xFF, int & 0xFF)
        case 8:
            (a, r, g, b) = ((int >> 24) & 0xFF, (int >> 16) & 0xFF, (int >> 8) & 0xFF, int & 0xFF)
        default:
            (a, r, g, b) = (255, 0, 0, 0)
        }
        self.init(
            .sRGB,
            red: Double(r) / 255,
            green: Double(g) / 255,
            blue: Double(b) / 255,
            opacity: Double(a) / 255
        )
    }
}

// MARK: - Adaptive Colors

extension ShapeStyle where Self == Color {
    static var cardBackground: Color {
        Color(.systemBackground)
    }

    static var secondaryBackground: Color {
        Color(.secondarySystemBackground)
    }

    static var tertiaryBackground: Color {
        Color(.tertiarySystemBackground)
    }
}
